sqlitecluster/SQLite.cpp

```diff
@@ -284,6 +285,20 @@ void SQLite::_sqliteLogCallback(void* pArg, int iErrCode, const char* zMsg) {
         // 17 is the length of "conflict at page" and the following space.
         const char* offset = strstr(zMsg, "conflict at page") + 17;
         _conflictPage = atol(offset);
+
+        // 17 is the length of "part of db table" and the following space.
+        const char* tableOffset = strstr(zMsg, "part of db table") + 17;
+
+        // Check if the tableOffset exists since not all conflicts are on tables
+        if (tableOffset) {
+            // Based on the SQLite log line, we should always have ';' after the table name,
+            // so let's use it to finish this loop.
+            int i = 0;
+            while (tableOffset[i] != ';') {
+                _conflictTable += tableOffset[i];
+                i++;
+            }
+        }
```

Comment:
I've confirmed this works as expected:

![image](https://github.com/user-attachments/assets/d0dd748b-003d-45b5-9bfd-07c3a921d87a)

Reply:
I'm using this log line as the base for this logic: https://github.com/Expensify/Bedrock/blob/676f804ed01ac09ffaedb2f3732069f0bbf50500/libstuff/sqlite3.c#L84720